src/color.scss

```diff
@@ -0,0 +1,100 @@
+// See LICENSE in the project root for license information.
+
+// ---------------------------------------------------------------------
+
+// Support for fallback values
+@import './../node_modules/@aurodesignsystem/design-tokens/dist/tokens/SCSSVariables.scss';
+
+/* stylelint-disable no-descending-specificity, declaration-empty-line-before */
+
+.popover {
+  border-color: var(--ds-auro-dropdown-popover-border-color);
+  background-color: var(--ds-auro-dropdown-popover-background-color);
+  box-shadow: -2px 0 5px 2px rgba(0 0 0 / .08), 0 2px 5px 1px rgba(0 0 0 / .08);
+  color: var(--ds-auro-dropdown-popover-text-color);
+}
+
+.label {
+  color: var(--ds-auro-dropdown-label-text-color);
+}
+
+.trigger {
+  border-color: var(--ds-auro-dropdown-trigger-border-color);
+  background-color: var(--ds-auro-dropdown-trigger-background-color);
+  color: var(--ds-auro-dropdown-trigger-text-color);
+
+  &:focus-within,
+  &:active {
+    --ds-auro-dropdown-trigger-border-color: var(--ds-color-border-active-default, #{$ds-color-border-active-default});
+  }
+
+  &:focus-within {
+    &:not(:active) {
+      --ds-auro-dropdown-trigger-border-color: var(--ds-color-ui-default-default, #{$ds-color-ui-default-default});
+      box-shadow: inset 0 0 0 1px var(--ds-auro-dropdown-trigger-border-color);
+    }
+  }
+}
+
+.helpText {
+  color: var(--ds-auro-dropdown-help-text-color);
+}
+
+:host([disabled]) {
+  --ds-auro-dropdown-trigger-text-color: var(--ds-color-text-ui-disabled-default, #{$ds-color-text-ui-disabled-default});
+  --ds-auro-dropdown-label-text-color: var(--ds-color-text-ui-disabled-default, #{$ds-color-text-ui-disabled-default});
+}
+
+:host([common]),
+:host([bordered]) {
+  .trigger {
+    --ds-auro-dropdown-trigger-border-color: var(--ds-color-border-secondary-default, #{$ds-color-border-secondary-default});
+
+    &:active,
+    &:focus-within {
+      --ds-auro-dropdown-trigger-border-color: var(--ds-color-border-ui-focus-default, #{$ds-color-border-ui-focus-default});
+    }
+  }
+}
+
+:host([error]) {
+  --ds-auro-dropdown-help-text-color: var(--ds-color-text-error-default, #{$ds-color-text-error-default});
+
+  .trigger {
+    --ds-auro-dropdown-trigger-border-color: var(--ds-color-border-error-default, #{$ds-color-border-error-default});
```

Comment:
Why set this when we aren't using a bordered trigger?

Reply:
The current design supports a red border when the dropdown is in an `error` state, even if the `border` attribute is not being used.

-https://www.figma.com/design/tSkHw1uJfDHy0Ml0aDp7Io/branch/SWOHKZpz2K78PPL43uvp5z/Auro-color-palette-(Variables)?node-id=4230-12845&m=dev